packages/transaction-controller/src/TransactionController.ts

```diff
@@ -1398,7 +1408,7 @@ export class TransactionController extends BaseControllerV1<
       getChainId: () => networkClient.configuration.chainId,
       getEthQuery: () => ethQuery,
       getTransactions: () => this.state.transactions,
-      isResubmitEnabled: true, // TODO: make this configurable
+      isResubmitEnabled: this.pendingTransactionOptions.isResubmitEnabled,
```

Comment:
I wonder if we'll want this to be configureable per networkClient/chainId but for now this is chill